### ISSUE TITLE
[Request]: Instantly turn off all settings that break missions 

### ISSUE THREAD

**Issue by BigboiKilljoy** [Yimura/YimMenu#2850]
### Problem

There's multiple settings in the menu that will break missions and it can be hard to keep track of them all, you might turn off 2 settings that are breaking a mission but then there's a third one still turned on that you forgot about.

### Solution

 Just have a button that checks for other settings and if a setting that breaks missions is on, it'll be turned off. In terms of turning those settings on again idk ill let you guys figure that out if you want to code this

### Reason

Doing a heist after turning off 2 mission breaking settings just to find out you still have some random setting breaking everything is annoying.

**Comment by Yimura** [comment]
Could you give a list of features that break missions? Perhaps we can make sure they’re only enabled when in freemode.

**Comment by BigboiKilljoy** [comment]
I only know about the ones you know about. For instance there's a setting called "Force Script Host" this will break missions and it tells you that when you hover your mouse over it.  These settings known for breaking missions should have a button that will turn them all off at once.

**Comment by maybegreat48** [comment]
> I only know about the ones you know about. For instance there's a setting called "Force Script Host" this will break missions and it tells you that when you hover your mouse over it. These settings known for breaking missions should have a button that will turn them all off at once.

Do you have anything else?

**Comment by BigboiKilljoy** [comment]
Are you guys asking me to test the menu and find settings that break missions?

**Comment by GreySideMM** [comment]
> Are you guys asking me to test the menu and find settings that break missions?

You said by yourself that you already know like 2-3 settings that break missions. So far you only named one. So we would assume you actually know more.. well since you made a issue just about that.
Keep in mind that other users mod menus can cause troubles too. Or when the host of the mission himself wants to cheat the mission. (I had the latter once, it blocked a Clear Ped Task in Cayo. I assume that was only due to cheating that mission by the host.)

**Comment by BigboiKilljoy** [comment]
> > Are you guys asking me to test the menu and find settings that break missions?
> 
> You said by yourself that you already know like 2-3 settings that break missions. So far you only named one. So we would assume you actually know more.. well since you made a issue just about that. Keep in mind that other users mod menus can cause troubles too. Or when the host of the mission himself wants to cheat the mission. (I had the latter once, it blocked a Clear Ped Task in Cayo. I assume that was only due to cheating that mission by the host.)

Yes but the settings i know about are ALREADY known. It literally says on the settings when you hover your mouse over them, that they will break missions.

**Comment by A5m0deu5** [comment]
I've noticed one: Doing casino heist when someone dies there is restart screen appears. Once all the players confirmed another try the defend notification appears: "Player tried invalid model crash you" and all players is stuck in loading screen. I'm the one Mod menu user there so i'm pretty sure problem is somewhere there. I've also tried to disable all guard features, reactions features. Maybe there is a passive crash defend that breaks a lobby.

**Comment by Sheeeeit** [comment]
Block `Force Teleport` can broke mission if it needed teleport you to the apartments for cutscene.
Same happens with `GTA Banner` - you can't complete heist prep delivery (Salvage Yard for example).
